src/win32.mak

```
@@ -288,12 +284,8 @@ tolf: $(GEN)\build.exe
 zip: detab tolf $(GEN)\build.exe
 	$(RUN_BUILD) $@
 
-scp: detab tolf $(MAKEFILES)
-	$(SCP) $(MAKEFILES) $(SCPDIR)/src
-	$(SCP) $(SRCS) $(SCPDIR)/src
-	$(SCP) $(GLUESRC) $(SCPDIR)/src
-	$(SCP) $(BACKSRC) $(SCPDIR)/src/backend
-	$(SCP) $(ROOTSRC) $(SCPDIR)/src/root
+scp: detab tolf $(GEN)\build.exe
```

No need to invoke `build.d` multiple times (and keep logic in this makefile).
```suggestion
scp: $(GEN)\build.exe
```
Add them as `deps`

---

I didn't want to add them as deps in the normal case, but I wanted to keep the windows dependencies the same for Walter (I think he's the only one who uses this).